test/js/computed-style-property-map.js

```diff
@@ -8,16 +8,8 @@ suite('ComputedStylePropertyMap', function() {
     document.documentElement.removeChild(this.element);
   });
 
-  test('ComputedStylePropertyMap is a ComputedStylePropertyMap and a ' +
-      'StylePropertyMap', function() {
-    var computedStyleMap = new ComputedStylePropertyMap(this.element);
-    assert.instanceOf(computedStyleMap, ComputedStylePropertyMap);
-    assert.instanceOf(computedStyleMap, typedOMTesting.StylePropertyMap);
-  });
-
-  test('ComputedStylePropertyMap.get works for properties that can only be ' +
-      'NumberValues', function() {
-    var computedStyleMap = new ComputedStylePropertyMap(this.element);
+  test('Computed StylePropertyMap.get works for properties that can be NumberValues', function() {
```

Comment:
Wording here is a little odd. This is testing a method of StylePropertyMapReadOnly, not StylePropertyMap.

Maybe
"Computed StylePropertyMapReadOnly.get works for properties that can only be NumberValues"
or
"Getting opacity from the computed style map works as expected"